### PR TITLE
Apply token manually on activity update request

### PR DESCRIPTION
**Pull request by mkuznyetsov**

### CODE REVIEW

File: core/codenvy-hosted-workspace-activity-agent/src/main/java/com/codenvy/workspace/WorkspaceActivityNotifier.java [hunk 1]
```diff
@@ -14,7 +14,10 @@
  */
 package com.codenvy.workspace;
 
+import com.google.common.annotations.VisibleForTesting;
```
Comment (sleshchenko) [inline core/codenvy-hosted-workspace-activity-agent/src/main/java/com/codenvy/workspace/WorkspaceActivityNotifier.java]: unused import?

**Comment by skabashnyuk** [comment]
ok

File: core/codenvy-hosted-workspace-activity-agent/src/main/java/com/codenvy/workspace/WorkspaceActivityNotifier.java (hunk 1 above)
Comment (mkuznyetsov) [inline core/codenvy-hosted-workspace-activity-agent/src/main/java/com/codenvy/workspace/WorkspaceActivityNotifier.java]: fixed

**Comment by akorneta** [comment]
ok

**Pull request merged**